playwright/integration/examples/hovering-toolbar.test.ts

```diff
@@ -35,15 +29,15 @@ test.describe('hovering toolbar example', () => {
       .locator('span[data-slate-string="true"]')
       .nth(0)
       .selectText()
-    await expect(page.locator('div').nth(12)).toHaveCSS('opacity', '1')
+    await expect(page.getByTestId('menu')).toHaveCSS('opacity', '1')
     await page
       .locator('span[data-slate-string="true"]')
       .nth(0)
       .selectText()
     await page
       .locator('div')
       .nth(0)
-      .click({ force: true })
-    await expect(page.locator('div').nth(12)).toHaveCSS('opacity', '0')
+      .click({ force: true, position: { x: 0, y: 0 } })
```

Comment:
without position prop in firefox it clicks exactly on toolbar, because it placed in the page center